src/js/components/EditsTableWrapper.jsx

```diff
@@ -102,7 +101,11 @@ EditsTableWrapper.propTypes = {
   code: PropTypes.number,
   syntacticalValidityEditsExist: PropTypes.bool,
   qualityVerified: PropTypes.bool,
-  macroVerified: PropTypes.bool
+  macroVerified: PropTypes.bool,
+  editsFetched: PropTypes.bool,
+  syntacticalValidityFetched: PropTypes.bool,
```

Comment:
Same as above, I don't see
- `syntacticalValidityFetched`,
- `qualityFetched`, or
- `macroFetched`

being used here.

In this case I don't see
- `base`,
- `code`,
- `syntacticalValidityEditsExist`,
- `qualityVerified`, or
- `macroVerified`

either. Those are probably just lingering from an old way of doing this.

Reply:
Same as above, except it's `Edits.jsx` which is getting passed the props which are in turn passed down to the table wrapper.